config/secrets.yml

```diff
@@ -0,0 +1,5 @@
+development:
+  secret_key_base: 
+  auth0_client_id: ve6NAufF1Pgm1jIhyQgEwa8PCG4vAoNF
+  auth0_client_secret: ynaE7yyYjvnuhFN_3bx435H2v0IF8jLQpwUCW8ovj-6ONRefuN7ker2L_RR5VfCI
+  auth0_domain: openware-barong.us.auth0.com
```

Comment:
you should do it on https://github.com/openware/barong/blob/master/config/initializers/barong.rb file